client/my-sites/earn/memberships/index.jsx

```diff
@@ -290,30 +317,36 @@ class MembershipsSection extends Component {
 	render() {
 		if ( this.props.isJetpackTooOld ) {
 			return (
-				<Notice
-					status="is-warning"
-					text={ this.props.translate(
-						'Please update Jetpack plugin to version 7.3 or higher in order to use the Membership button block'
-					) }
-					showDismiss={ false }
-				>
-					<NoticeAction
-						href={ `https://wordpress.com/plugins/jetpack/${ this.props.siteSlug }` }
-						icon="external"
-					/>
-				</Notice>
+				<div>
+					{ this.renderOnboarding() }
+					<Notice
+						status="is-warning"
+						text={ this.props.translate(
+							'Please update Jetpack plugin to version 7.4 or higher in order to use the Recurring Payments button block'
```

Comment:
"Please update the Jetpack plugin [...]"

Reply:
Fixed!